### ISSUE TITLE
Get Parent Node Of the dropped Node

### ISSUE THREAD

**Issue by medicareamaze** [phphe/he-tree-vue#74]
I am trying to identify the **Parent Node** of the dropped node. 

```
<Tree ref="tree" @drop="ondrop">
</Tree>

ondrop(store) {
      var parent = this.tree.getNodeParentByPath(store.targetPath)
      console.log(parent)
}
```
It looks like store.targetPath is still referencing the original node. I tried your example from another issue posted in  [https://github.com/phphe/he-tree-vue/issues/65]

`let t = store.targetPath.slice(0)
  t[t.length - 1]++
  console.log(store.targetTree.getNodeByPath(t));`

It still does not work. I just need to get the Parent ID of the node and my project will be complete. This is now becoming a show stopper. Any help is really appreciated

**Comment by phphe** [comment]
The issue has been fixed since he-tree-vue@3.1.0. Check https://github.com/phphe/he-tree-vue/issues/20 

This example is deprecated. https://github.com/https://github.com/phphe/he-tree-vue/issues/65

**Comment by medicareamaze** [comment]
Thanks! It works!
